packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js

```diff
@@ -75,3 +111,59 @@ exports.howYouNotifiedPeople = (answers) => {
 	}
 	return notifiedPeople;
 };
+
+/**
+ * @param {FullAppellantSubmission} users
+ * @returns {DataModelUsers}
+ */
+exports.formatApplicationSubmissionUsers = ({
+	isAppellant,
+	appellantFirstName,
+	appellantLastName,
+	contactFirstName,
+	contactLastName,
+	Appeal: {
+		Users: [
+			{
+				AppealUser: { email }
+			}
+		]
+	}
+}) => {
+	/** @type {DataModelUsers} */
+	const users = [
+		{
+			salutation: null,
+			firstName: appellantFirstName,
+			lastName: appellantLastName,
+			emailAddress: null,
+			serviceUserType: 'Appellant'
```

Comment:
think the default should be contactFirstName + contactLastName + the email
type becomes isAppellant ? 'Appellant' : 'Agent'

Then a second record for appellant is added if !appellant
```js
if (!isAppellant) {
		users.push({
			salutation: null,
			firstName: appellantFirstName,
			lastName: appellantLastName,
			emailAddress: null,
			serviceUserType: 'Appellant'
		});
	}
```